packages/vscode-extension/src/project/metro.ts

```diff
@@ -376,7 +376,8 @@ export class Metro implements Disposable {
       (page) =>
         page.reactNative &&
         (page.title.startsWith("React Native Bridge") ||
-          page.description.endsWith("[C++ connection]"))
+          page.description.endsWith("[C++ connection]") ||
+          page?.reactNative?.capabilities?.prefersFuseboxFrontend)
```

Comment:
These shouldn't be necessary, we've already checked for `page.reactNative` being truthy.
```suggestion
          page.reactNative.capabilities?.prefersFuseboxFrontend)
```